In [9]:
import pandas as pd
arr = pd.read_csv("arrestdata.tsv", sep="\t")

In [13]:
count = {}
for index, row in arr.iterrows():
    name = row["INCIDENTNEIGHBORHOOD"]
    qty = 1
    if name in count:
        count[name] = count[name]+qty
    else:
        count[name] = qty
print(count)


{'Bloomfield': 984, 'Outside City': 316, 'Westwood': 226, 'East Allegheny': 2217, 'Crafton Heights': 601, 'Greenfield': 537, 'Brookline': 1390, 'Regent Square': 37, 'East Liberty': 1889, nan: 2489, 'East Hills': 1194, 'Carrick': 2352, 'Allegheny Center': 1244, 'South Side Flats': 3497, 'Sheraden': 1285, 'East Carnegie': 48, 'Golden Triangle/Civic Arena': 83, 'Point Breeze North': 289, 'Perry South': 1057, 'Allentown': 947, 'North Shore': 814, 'Marshall-Shadeland': 1516, 'Central Oakland': 521, 'Middle Hill': 932, 'Strip District': 503, 'Lincoln-Lemington-Belmar': 1362, 'Central Northside': 23, 'Troy Hill-Herrs Island': 6, 'Highland Park': 558, 'Homewood North': 1969, 'Northview Heights': 713, 'Brighton Heights': 1114, 'Hazelwood': 1396, 'Perry North': 587, 'Bedford Dwellings': 665, 'Elliott': 663, 'Beechview': 1201, 'Fineview': 796, 'Glen Hazel': 169, 'Spring Hill-City View': 711, 'Bluff': 1502, 'Manchester': 471, 'Homewood West': 764, 'Outside County': 57, 'Point Breeze': 301, 'Homewo

__[full list here](https://docs.google.com/document/d/1R1nbctVSxgrlZUJC-xuTfA5c192NnIaROhn-2pDP7bE/edit?usp=sharing)__

**The neighborhood with the most arrests is:** 
South Side Flats - 3497

**The neighborhood with the least is:**
Central North Side - 23

**Some of the more well-known Pittsburgh neighborhoods include:**
 - South Side - 3497
 - Carrick - 2352
 - East Liberty - 1889
 - Mount Washington - 1434
 - Hazelwood - 1396
 - Brookline - 1390
 - Beechview - 1201
 - Bloomfield - 984
 - North Shore - 814
 - Elliott - 663
 - Greenfield - 537
 - Central Oakland - 521
 - Strip district - 503
 - Troy Hill - 437
 - West End - 290
 - Mount Oliver - 144
 - Homestead - 39

For my code, it creates a list of all of the different neighbohoods in the data table. Then, it adds 1 to the count for each time that the neighborhood is repeated in the table. After that it prints the list out.

From the data we can see that the top 5 places with the least arrests are:
 - Central Northside (23)
 - Regent Square (37)
 - Ridgemont (37)
 - Homestead (39)
 - Chartiers City (46)
 - East Carnegie (48)
 - St. CLair (55)
 - Golden Triangle/Civic Arena (83)
 - Allegheny West (130)
 - Mount Oliver (144)